packages/roosterjs-editor-core/lib/coreApi/selectTable.ts

```diff
@@ -0,0 +1,202 @@
+import { getTagOfNode, toArray } from 'roosterjs-editor-dom';
+import {
+    EditorCore,
+    SelectionRangeTypes,
+    TableSelection,
+    SelectTable,
+    Coordinates,
+} from 'roosterjs-editor-types';
+
+const TABLE_ID = 'tableSelected';
+const CONTENT_DIV_ID = 'contentDiv_';
+
+let styleElement: HTMLStyleElement;
```

Comment:
having a global reference of STYLE node can be a problem when there is projection popout, because each window need its own STYLE node, but a global one stops creating more STYLE node for child windows. And currently the code will remove all style rules from the STYLE tag when unselect, that will also remove rules from other editor in the same window